yarn-project/aztec/src/cli/cmds/start_pxe.ts

```diff
@@ -1,36 +1,21 @@
 import type { NamespacedApiHandlers } from '@aztec/foundation/json-rpc/server';
 import type { LogFn } from '@aztec/foundation/log';
-import {
-  type CliPXEOptions,
-  type PXEService,
-  type PXEServiceConfig,
-  allPxeConfigMappings,
-  createPXEService,
-} from '@aztec/pxe/server';
+import { type CliPXEOptions, type PXEServiceConfig, allPxeConfigMappings, createPXEService } from '@aztec/pxe/server';
 import { type AztecNode, PXESchema, createAztecNodeClient } from '@aztec/stdlib/interfaces/client';
 import { makeTracedFetch } from '@aztec/telemetry-client';
+import { TestWallet } from '@aztec/test-wallet';
 
 import { extractRelevantOptions } from '../util.js';
 import { getVersions } from '../versioning.js';
 
-export type { PXEServiceConfig, CliPXEOptions };
+export type { CliPXEOptions, PXEServiceConfig };
 
-export async function startPXE(
+export async function startPXEServicesGetWallet(
```

Comment:
We now always just used the pxe to instantiate a test wallet so I just return a wallet from this function. This will get nuked in a followup PR when PXE will stop being a service.